WordPress/Classes/ViewRelated/Site Creation/Final Assembly/SiteAssemblyWizardContent.swift

```diff
@@ -139,19 +139,20 @@ final class SiteAssemblyWizardContent: UIViewController {
                 SiteCreationAnalyticsHelper.trackSiteCreationSuccess(self.siteCreator.design)
             }
             if status == .succeeded,
-               shouldPerformDomainPurchasingStep,
+               shouldPerformPurchasingStep,
                let domain = self.siteCreator.address,
+               let planId = self.siteCreator.planId,
                let blog = self.createdBlog {
-                self.attemptDomainPurchasing(domain: domain, site: blog)
+                self.attemptPlanAndDomainPurchasing(domain: domain, planId: planId, site: blog)
             } else {
                 self.contentView.status = status
             }
         }
     }
 
-    /// The site must be created before attempting domain purchasing.
-    private func attemptDomainPurchasing(domain: DomainSuggestion, site: Blog) {
-        self.domainPurchasingController.purchase(domain: domain, site: site) { [weak self] result in
+    /// The site must be created before attempting domain and plan bundle purchasing.
+    private func attemptPlanAndDomainPurchasing(domain: DomainSuggestion, planId: Int, site: Blog) {
```

Comment:
nit: Is this checkout web view shown to users who choose a free domain (e.g. xyz.wordpress.com`) and a paid plan? If so, it might be more descriptive to not mention "plan and domain" here since it could be a plan and domain bundle OR just a plan.

Reply:
Correct. Yes, it will be shown to users who choose a free domain and paid plan. I will try to make it clearer.